arduino/scout/scout.ino

```diff
@@ -67,17 +67,14 @@ void updateFromAnalogInputs() {
 }
 
 void loop() {
-  uint8_t i;
-  uint8_t size;
-
   buffer.populate();
-  size = buffer.getSize();
+  uint8_t size = buffer.getSize();
 
   // TODO: do this less often...
   updateFromAnalogInputs();
 
   if (printToSerial) {
-    for (i = 0; i < size; i++) {
+    for (uint8_t i = 0; i < size; i++) {
```

Comment:
Lizard embedded firmware brain again -- I try to declare variables only once and reuse them, which is why I did it the way I did it, but this loop is not timing-critical so the change is fine by me. Just wanted to explain the reasoning.